osidb/models.py

```diff
@@ -1296,6 +1322,31 @@ def _validate_tracker_flaw_accesses(self):
                 "Tracker is public but is associated with an embargoed flaw."
             )
 
+    def _validate_notaffected_open_tracker(self):
+        """
+        Check whether notaffected products have open trackers.
+        """
+        affect = self.affects.filter(
+            affectedness=Affect.AffectAffectedness.NOTAFFECTED
+        ).first()
+
+        if not self.is_closed and affect:
+            raise ValidationError(
+                f"Affect for {affect.ps_module}/{affect.ps_component} is marked as "
```

Comment:
This is kinda nitpick but I am thinking about the situation that the tracker belongs to multiple affects and currently (may change in the future) it means that all of these affects have the same PS module and PS component but they belong to different flaws. Therefore PS module and PS component would not be enough to uniquely identify the bogus affect. We could either add its UUID or the related flaw UUID.

Reply:
Makes sense, added affect's uuid